OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    </incipit>
+                                    
+                                    <explicit xml:lang="gez">
+                                        <locus from="55va" to="55vb"/>
+                                        ወገብረ፡ ምጽዋተ፡ በዙኃ፡ ለነዳያን፡ ወለምስኪናን፡ ወለቤተ፡ ክርስቲያን፡ ወኀቤሃ<hi rend="rubric">፨</hi> ዘይደሉ<hi rend="rubric">፨</hi> 
+                                        ወሶበ፡ ርእየት፡ እግዝእትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም</hi> 
+                                        ወላዲተ፡ አምላክ፡ ቲብዓተ፡ ልቡ፡ ተመልሐ፡ ሐፅ፡ እምዓይኑ፡ ወእምዓይነ፡ ምስል፡ ዘሰምዕ፡ በአሐቲ፡ ሌሊት<hi rend="rubric">፨</hi> 
+                                        ወገብአት፡ ዓይኑ፡ ከመ፡ ካልዕታ<hi rend="rubric">፨</hi>  ወሶበ፡ ርእዩ፡ ሕዝብ፡ ዓይነ፡ ሥዕል፡ አንከሩ፡ ተአምሪሃ፡
+                                        ለእግዝእ<cb n="b"/>ትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም፡</hi> 
+                                        ወላዲተ፡ አምላክ<hi rend="rubric">፨</hi> 
+                                    </explicit>
+                                    
+                                    <explicit type="supplication">
+                                        <locus target="#55vb"/>
+                                        ጸሎታ፡ ወበረከታ፡ ወምሕረተ፡ ፍቁር፡ ወልዳ፡ የሃሉ፡ ምስለ፡ ገብራ፡ 
+                                        <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ 
+                                            ክርስቶስ<supplied reason="omitted" resp="DR">፡</supplied></hi></persName> ለዓለመ፡
+                                        ዓለም፡ አሜን<hi rend="rubric">፨</hi>
+                                    </explicit>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i3.2.16.2">
+                                    <locus target="#55vb"/>
+                                    <title type="complete" ref="LIT4274AkkonuB#Stanza15"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i3.2.17">
+                                <locus from="56ra" to="58ra"/>
+                                <title type="complete" ref="LIT5673AkkonuColl#AkkColl16"/>
+                                <textLang mainLang="gez"/>
+                                
+                                <msItem xml:id="ms_i3.2.17.1">
+                                    <locus from="56ra" to="57vb"/>
+                                    <title type="complete" ref="LIT3602Miracle"/>
+                                    <textLang mainLang="gez"/>
+                                    
+                                    <incipit type="inscriptio" xml:lang="gez">
+                                        <locus target="#56ra"/>
+                                        <hi rend="rubric">ተአምሪሃ፡ ለእግዝእ</hi>ትነ፡ ቅድስት፡ ድን <hi rend="rubric">ግል፡ በ፪ ማርያም፡
```

Comment:
no space in `ድን<hi rend="rubric">ግል፡`. I also think there should be no space in በ፪ማርያም፡